README.rst

```diff
@@ -32,7 +32,7 @@ Status shields
     :alt: Benchmarks
 
 * .. image:: https://readthedocs.org/projects/gammapy/badge/?version=latest
-    :target: https://readthedocs.org/projects/gammapy/?badge=latest
+    :target: https://readthedocs.io/projects/gammapy/?badge=latest
```

Comment:
also I see that you have a gammapy.org domain, so I'm changing all the gammapy.readthedocs to docs.gammapy.org.